### DISCUSSION TITLE
After the plugin anuvyklack/fold-preview.nvim is configured with event = "VeryLazy", the l key will fail

### DISCUSSION THREAD

**Discussion by binary4cat** [LazyVim/LazyVim#176] [General]
After installing the `anuvyklack/fold-preview.nvim` plugin, if you set `event = "VeryLazy"`, it will cause the `l` key to trigger the `:Lazy` command. I found that the `l` key triggers `:Lazy` is It is configured in the plugin `goolord/alpha-nvim`, but I don’t know why it is triggered in `anuvyklack/fold-preview.nvim`. I suspect it is caused by the plug-in `anuvyklack/keymap-amend.nvim` it depends on. But I didn't check carefully, I will look carefully and come back. 

My configuration is [here](https://github.com/binary4cat/.config/blob/306e8b58d90e5cb1895dc825b03e980f520cbc82/nvim/lua/plugins/editor.lua#L83).

**Comment by loiccoyle** [comment]
I haven't looked into any of the plugins you mentioned, but the `<leader>l` mapping for `Lazy` is defined here:

https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/config/keymaps.lua#L89

**Reply by binary4cat** [reply]
The single key `l` is defined [here](https://github.com/LazyVim/LazyVim/blob/663e142bcd99e5546989b762ef4d04680d8715a2/lua/lazyvim/plugins/ui.lua#L234) , but it's not clear why it was affected by another plugin.

And if you go in from the dashboard, there will be a problem, if you open the file directly, there will be no problem:
```shell
# key "l" disorder
nvim

# There will be no confusion about the "l" key
nvim ~/.config/nvim/init.lua
```

**Reply by loiccoyle** [reply]
Oh I see, maybe you can try to not load `anuvyklack/fold-preview.nvim` in the `alpha` filetype?

Something like:
```lua
return {
  "anuvyklack/fold-preview.nvim",
  lazy = true,
  dependencies = { "anuvyklack/keymap-amend.nvim" },
  init = function()
    vim.api.nvim_create_autocmd("FileType", {
      pattern = { "*" },
      callback = function()
        if not vim.bo.ft == "alpha" then
          require("lazy").load({ plugins = { "fold-preview.lua" } })
        end
      end,
    })
  end,
}

```

**Reply by binary4cat** [reply]
Awesome, all functions are perfect with your solution, thanks!

**Comment by folke** [comment]
That `l` keymap only exists on the alpha dashboard. not sure why that plugin you mention use that mapping...
